### PR TITLE
Adding in page extensions integration tests

### PR DESCRIPTION
**Pull request by davidfateh**
# Purpose of PR
This adds page extension integration tests. This is blocked by #212 

## PR Checklist

- [x] Tests are added/updated/not required
- [x] Tests are passing
- [x] Typescript typings are added/updated/not required

### CODE REVIEW

File: test/extensions/test-page-extension/package.json [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  "private": true,
+  "devDependencies": {
+    "@babel/core": "7.3.4",
+    "@babel/plugin-proposal-class-properties": "7.3.4",
+    "@babel/plugin-transform-runtime": "7.3.4",
+    "@babel/preset-env": "7.3.4",
+    "@babel/preset-react": "7.0.0",
+    "@contentful/contentful-extension-scripts": "0.9.0",
+    "cssnano": "4.1.10",
+    "contentful-cli": "0.26.6",
```
Comment (suevalov) [inline test/extensions/test-page-extension/package.json]: would be nice to update all contentful packages to latest available on npm.

File: test/integration/tasks/create-configuration-files.js [hunk 2]
```diff
@@ -4,18 +4,21 @@ const printStepTitle = require('../utils').printStepTitle
 
 module.exports = async ({ cmaToken, spaceId, environmentId }) => {
   printStepTitle('Creating configuration files based on environment variables')
+  async function writeJSONForExtension(extensionId) {
+    await writeJSONFile(resolvePath(`test/extensions/${extensionId}/.contentfulrc.json`), {
+      cmaToken: cmaToken,
+      activeSpaceId: spaceId,
+      activeEnvironmentId: environmentId
+    })
+    await writeJSONFile(resolvePath('cypress.env.json'), {
```
Comment (suevalov) [inline test/integration/tasks/create-configuration-files.js]: It should be written once. Not related to extension.

Comment (davidfateh) [reply test/integration/tasks/create-configuration-files.js]: I went through and made sure I wasn't rewriting anything else either.

File: test/cypress/integration/field-extension.spec.js [hunk 3]
```diff
@@ -24,4 +24,23 @@ context('Field extension', () => {
...
+
+    cy.get('[data-test-id="page-extension"] iframe').captureIFrameAs('extension')
+
+    cy.get('@extension')
+      .find('[data-test-id="cf-ui-page-extension"]')
+      .should('exist')
+
+    cy.get('@extension')
+      .find('button:first')
+      .click()
```
Comment (suevalov) [inline test/cypress/integration/field-extension.spec.js]: would be cool to verify that `path` is passed to `invocation` params. we could check the output of `JSON.stringify` output

Comment (davidfateh) [reply test/cypress/integration/field-extension.spec.js]: Yeah I think that's a good idea, I can add that in.

Comment (davidfateh) [reply test/cypress/integration/field-extension.spec.js]: On second thought, I will skip this for now. We should have tests for the method.

**Pull request merged**